runtime/karura/src/lib.rs

```diff
@@ -1551,6 +1553,20 @@ impl module_idle_scheduler::Config for Runtime {
 	type MinimumWeightRemainInBlock = MinimumWeightRemainInBlock;
 }
 
+parameter_types! {
+	pub WormholeAUSDCurrencyId: CurrencyId = CurrencyId::Erc20(EvmAddress::from(hex_literal::hex!["0000000000000000000100000000000000000001"]));
```

Comment:
Create an issue and update here after creating waUSD

Reply:
And disable it here:
https://github.com/AcalaNetwork/Acala/blob/bf3d860f388117b8bb74502795be38704fdacfcd/runtime/karura/src/lib.rs#L213-L220

Reply:
Created an issue here: 
https://github.com/AcalaNetwork/Acala/issues/1967

Reply:
Issue created and HonzonBridge call disabled